services/frontend-service/src/ui/Batch.tsx

```diff
@@ -56,9 +68,55 @@ export interface ConfirmationDialogProviderProps {
 const InCart = (actions: BatchAction[], action: BatchAction) =>
     actions ? actions.find((act) => JSON.stringify(act.action) === JSON.stringify(action.action)) : false;
 
+const IsDeployment = (t: ActionTypes) =>
+    t === ActionTypes.Deploy || t === ActionTypes.PrepareUndeploy || t === ActionTypes.Undeploy;
+
+const IsConflictingAction = (cartActions: BatchAction[], newAction: BatchAction) => {
```

Comment:
Do you mean that the user can `replace` the actions in the cart? 
I think that's a good idea, though I think it could take some time to be done. because I'm not sure yet how to delete the actions and if there are multiple conflicts. and also if you can return them as references to the actual actions or indices? 

I can create a new story. would that work?

Reply:
Yeah sure!